policy/Makefile

```diff
@@ -9,6 +9,7 @@ RHEL_VERS=$(shell echo $(DISTRO) | sed 's/RHEL//')
 
 POL_VERS := $(shell $(CHECKPOLICY) -V |cut -f 1 -d ' ')
 MOD_POL_VERS := $(shell $(CHECKMODULE) -V |cut -f 2 -d '-')
+MAX_KERNEL_POLICY := $(shell sestatus | grep "Max kernel policy" | awk 'END {print $$NF}')
```

Comment:
Could just read /path/to/selinuxfs/policyvers as is done in tests/ioctl/test script to determine if the kernel supports it.

Reply:
OK, I can change it.